volume_scroller@francislavoie.github.io/extension.js

```diff
@@ -102,22 +85,17 @@ class VolumeScroller
         return Clutter.EVENT_STOP;
     }
 
-    _handle_sink_change(controller, id)
-    {
+    _handle_sink_change(controller, id) {
         this.sink = controller.lookup_stream_id(id);
     }
 
-    _show_volume(volume)
-    {
+    _show_volume(volume) {
         const percentage = volume / this.volume_max;
         let n;
 
-        if (volume === 0)
-        {
+        if (volume === 0) {
             n = 0;
-        }
-        else
-        {
+        } else {
             n = parseInt(3 * percentage + 1);
             n = Math.max(1, n);
             n = Math.min(3, n);
```

Comment:
I think we can use https://gjs.guide/extensions/upgrading/gnome-shell-45.html#math-clamp `Math.clamp()` here to simplify this slightly

```js
// If muted, use muted icon, otherwise clamp to the nearest icon in thirds
const iconIndex = volume === 0
    ? 0
    : Math.clamp(parseInt(3 * percentage + 1), 1, 3)
```